core/src/main/scalajs/weaponregex/WeaponRegeXJS.scala

```diff
@@ -36,7 +38,7 @@ object WeaponRegeXJS {
     *   A JavaScript Array of [[weaponregex.model.mutation.Mutant]] if can be parsed, or throw an exception otherwise
     */
   @JSExportTopLevel("mutate")
-  def mutate(pattern: String, options: MutationOptions = new MutationOptions): js.Array[MutantJS] = {
+  def mutate(pattern: String, flags: String, options: MutationOptions): js.Array[MutantJS] = {
```

Comment:
Is this change in API on purpose? These no longer have default arguments

Reply:
git diff got confused 😵 the exact same old API method should just be right below this one.

Reply:
Ah I see. Could we still have the `MutationOptions` as a default arg? And I think preferably the flags as well

Reply:
Scala doesn’t like multiple overloaded alternatives of method with default arguments, so I chose to preserve the old API. Plus, I think it’s logical for the user to explicitly define the regex flavor in mutation options when they want to use String flags, since not all flavors implement flags as a String.